src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocpackage/Example2.java

```diff
@@ -0,0 +1,17 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MissingJavadocPackage"/>
+  </module>
+</module>
+*/
+// xdoc section -- start
+/*
+ * Block comment is not a javadoc
+ */
```

Comment:
violation is not raised for the above block comment. @romani can you please help and provide some insight.

Reply:
https://checkstyle.org/checks/javadoc/missingjavadocpackage.html#Description

>Checks for missing package definition Javadoc comments in package-info.java files.

Please rename file.

Reply:
All files should be named `package-info.java` and located in different packages/folders.

There should be no `class` in such files, so please add such files to suppressions

Reply:
Done